cmd/datamon/cmd/cli_fuse_test.go

```diff
@@ -206,7 +206,7 @@ func testMountCmdParams(t testing.TB, testType, repo, pathToMount, pathBackingFs
 	}
 }
 
-func testMountReady(t testing.TB, token string, defaultWait time.Duration) func(*zap.Logger, io.Reader) {
+func makeBlockingTestWaiterForReader(t testing.TB, token string, defaultWait time.Duration) func(*zap.Logger, io.Reader) {
```

Comment:
I think we are in a contention here. Please keep the short names. Since you come with a more general use case for that func, I am suggesting `testWaitForReader` as a compromise.

At any rate  `makeMyselfJustTryOnceToTestIfWaitingForSomeReaderEventuallyWorksKnowingThatSaidReaderMightSometimesNotBeFunctionningProperly` is not something I would like to generalize.

Reply:
actually, i recalled your comment on my request for a multi-word name on the response to Kevin's perf. improvement request, and chose a name you'd definitely object to ... probably should've left a comment.

of course the name needed to be changed (not only about `Mount`), so i'll go with the `testWaitForReader`, np.